src/core/components/highlight-code.jsx

```diff
@@ -39,6 +39,18 @@ export default class HighlightCode extends Component {
     }
   }
 
+  componentDidMount() {
+    [this.syntaxHighlighter, this.pre]
+    .filter(element => !!element)
+    .map(element => element.addEventListener("mousewheel", this.preventYScrollingBeyondElement, { passive: false }))
```

Comment:
```suggestion
    .map(element => element?.addEventListener("mousewheel", this.preventYScrollingBeyondElement, { passive: false }))
```
Could be shortened one more time.